### ISSUE TITLE
MLPAttack2021 showing wrong accuracy while early stopping on validation loss.

### ISSUE THREAD

**Issue by AranyaiitrGit** [nils-wisiol/pypuf#189]
I wish to print the accuracy of the MLPAttack2021 model even after early stopping occurs. Early stopping on validation loss is implemented to prevent overfitting, yet I'm uncertain about obtaining the prediction accuracy.

For instance, in modeling an LFSR operating on an Arbiter PUF, I am examining the resilience of LFSR to an MLP Attack. When using the CRP dataset, the training halts after a few epochs, but then, how do I determine the accuracy of that attack?

![lfsr](https://github.com/nils-wisiol/pypuf/assets/95477786/b4a364b9-2989-4d7d-80cf-3962dcd7df8a)
![image](https://github.com/nils-wisiol/pypuf/assets/95477786/d1078cb9-d3a4-491c-ad85-a1aef37ee008)

**Comment by AranyaiitrGit** [comment]
>>> import pypuf.metrics
>>> pypuf.metrics.similarity(puf, model, seed=4)
array([0.97])


Which accuracy we will get from this 'simirality' metrics ?

**Comment by nils-wisiol** [comment]
The documentation for main currently has a problem and does not show the details on the accuracy metrics. Here is a temporary fix with that information: https://pypuf.readthedocs.io/en/fix-docs/metrics/distance.html

**Comment by nils-wisiol** [comment]
The update has been pushed to the regular docs page, too.
